docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+ 
+![PR welcome](https://img.shields.io/github/labels/neurobagel/planning/PR%20welcome) 
+*Issue that is not an internal priority, but external pull requests to address it are welcome.*
+
+![quick fix](https://img.shields.io/github/labels/neurobagel/planning/quick%20fix): 
+*Issue that should involve minimal planning or implementation work, given an understanding of the relevant code.*
+
+## Making a change
+
+All Neurobagel issues are expected to be addressed through [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+
+As an external contributor, the process you would follow to make your [proposed changes](#identifying-an-issue-to-work-on) should look something like this:
+
+#### 1. [Fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) the relevant Neurobagel repository to your profile.
+
+#### 2. [Clone](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#cloning-your-forked-repository) your fork of the Neurobagel repository to your local machine.
+
+To keep up with changes in the Neurobagel repository while you work and avoid merge conflicts later on, make sure to:
+
+- [Add the "upstream" Neurobagel repository as a remote](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository) to your locally cloned repository
+- [Keep your fork up to date](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-command-line) with the upstream repository
+
+#### 3. Set up a development environment.
+Refer to the repository README for instructions on how to set up a development environment so you can test any local code changes you make.
+Note that the steps to set up a development environment (usually under a "Local installation", "Manual installation", or "Development environment" section) are generally different than those used to install the tool purely as a user, which usually appears at the top of the README.
+
+For example, for the [Neurobagel CLI](https://github.com/neurobagel/bagel-cli):
+
+- ["Development" mode installation steps](https://github.com/neurobagel/bagel-cli?tab=readme-ov-file#installation)
+- [Normal (non-development mode) installation steps](https://github.com/neurobagel/bagel-cli?tab=readme-ov-file#development-environment)
+
+If you are working on a repository written in Python, we recommend that you also set up automatic code formatting and linting according to the project's code style, 
+which is pre-configured in the development environment for these repositories using [pre-commit](https://pre-commit.com/).
+
+Run the following from the repository root of your local clone:
+```bash
+pre-commit install
+```
```

Comment:
it'd be `npx husky init` for the JS codebase. But that's in the README as well. @rmanaem wdyt: add that here as well or leave up to the repo?

Reply:
I think it's better to leave it up to the repo.